contract/src/orca.proposal.js

```diff
@@ -1,13 +1,31 @@
 import { E } from '@endo/far';
 import { installContract } from './platform-goals/start-contract.js';
 import { makeTracer } from './tools/debug.js';
+// import '@agoric/orchestration/src/types.js';
+// import '@agoric/vats/src/types.js';
+// import '@agoric/vats/src/localchain.js';
+// import '@agoric/vow/src/types.js';
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+// import { atomicTransfer } from '@agoric/zoe/src/contractSupport/index.js';
+/// <reference types="@agoric/zoe/src/contractFacet/types-ambient"/>
 
 /**
  * @import {ERef} from '@endo/far';
  * @import {BootstrapManifest} from '@agoric/vats/src/core/lib-boot.js';
+ * @import {OrchestrationPowers} from './exos/cosmos-interchain-service.js';
+ * @import {Remote} from '@agoric/vow';
  * @import {OrcaSF} from './orca.contract.js';
  */
 
+/**
+ * * @typedef {import('./orca.contract.js').OrcaSF} OrcaSF
+ */
+
+/**
+ * @typedef {import('@agoric/vow').Remote<import('@agoric/vats/src/localchain.js').LocalChain>} LocalChain
```

Comment:
```suggestion
 * @import { LocalChain } from '@agoric/vats/src/localchain.js';
```

Then use `Remote<LocalChain>`.